runtime/state.go

```diff
@@ -0,0 +1,143 @@
+package runtime
+
+import (
+	"fmt"
+
+	"github.com/apex/log"
```

Comment:
I think this is a mistaken import. I believe it should be log `"github.com/sirupsen/logrus"`

Remove the apex log, and add the logrus UNDER the clients, like it looks below. This makes goimports happy and uses the correct logger.

```golang
import (
    "fmt"

    "github.com/automationbroker/bundle-lib/clients"
    log "github.com/sirupsen/logrus"
    "k8s.io/api/core/v1"
    kerror "k8s.io/apimachinery/pkg/api/errors"
    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
)
```